packages/fluentui/docs/src/examples/components/Datepicker/Types/DatepickerExampleLocalizationStrings.shorthand.tsx

```diff
@@ -1,11 +1,11 @@
 import { Datepicker } from '@fluentui/react-northstar';
 import * as React from 'react';
 
-const DatepickerExampleLocalizationStrings = () => (
-  <Datepicker
-    shortDays={['أحد', 'اثنين', 'ثلاثاء', 'أربعاء', 'خميس', 'جمعة', 'سبت']}
-    days={['أحد', 'اثنين', 'ثلاثاء', 'أربعاء', 'خميس', 'جمعة', 'سبت']}
-    months={[
+const DatepickerExampleLocalizationStrings = () => {
+  const localizationStrings = {
+    shortDays: ['أحد', 'اثنين', 'ثلاثاء', 'أربعاء', 'خميس', 'جمعة', 'سبت'],
```

Comment:
Just a thought, would be nice if all these props in the `Datepicker` are ending with `String` or `Label` suffix, so it would be obvious they are just strings. to be honest `goToToday` for example is not associating me that it is a string.

Reply:
That's a great point! 

We are actually having further discussions about whether the `localizationStrings` should be flattened in the DatepickerProps or not. Currently, they are flattened, so within the ICalendarStrings, it might make sense to not have the suffix but in DatepickerProps, it actually becomes confusing.

I propose to bring up this issue again in the planned v1 task to revisit the approach of localizationStrings. 

WDYT? 

FYI: @pompomon @layershifter